manifests/mod/passenger.pp

```diff
@@ -963,26 +731,20 @@
   # - $passenger_use_global_queue : since 2.0.4.Deprecated in 4.0.0.
   # - $passenger_user : since 4.0.0.
   # - $passenger_user_switching : since 3.0.0.
-  # - $rack_auto_detect : since unkown. Deprecated in 4.0.0.
-  # - $rack_base_uri : since unkown. Deprecated in 3.0.0.
   # - $rack_env : since 2.0.0.
-  # - $rails_allow_mod_rewrite : since unkown. Deprecated in 4.0.0.
-  # - $rails_app_spawner_idle_time : since unkown. Deprecated in 4.0.0.
-  # - $rails_auto_detect : since unkown. Deprecated in 4.0.0.
-  # - $rails_base_uri : since unkown. Deprecated in 3.0.0.
-  # - $rails_default_user : since unkown. Deprecated in 3.0.0.
   # - $rails_env : since 2.0.0.
   # - $rails_framework_spawner_idle_time : since unkown. Deprecated in 4.0.0.
-  # - $rails_ruby : since unkown. Deprecated in 3.0.0.
-  # - $rails_spawn_method : since unkown. Deprecated in 3.0.0.
-  # - $rails_user_switching : since unkown. Deprecated in 3.0.0.
-  # - $wsgi_auto_detect : since unkown. Deprecated in 4.0.0.
   # - $rails_autodetect : this options is only for backward compatiblity with older versions of this class
   # - $rack_autodetect : this options is only for backward compatiblity with older versions of this class
```

Comment:
Does that mean it can be removed?

Reply:
initially I kept it for reference but yeah it can be removed..

Reply:
done